contracts/cw-core/src/contract.rs

```diff
@@ -160,19 +164,14 @@ pub fn execute_admin_msgs(
 ) -> Result<Response, ContractError> {
     let admin = ADMIN.load(deps.storage)?;
 
-    match admin {
-        Some(admin) => {
-            // Check if the sender is the DAO Admin
-            if sender != admin {
-                return Err(ContractError::Unauthorized {});
-            }
-
-            Ok(Response::default()
-                .add_attribute("action", "execute_admin_msgs")
-                .add_messages(msgs))
-        }
-        None => Err(ContractError::NoAdmin {}),
+    // Check if the sender is the DAO Admin
+    if sender != admin {
```

Comment:
Does raw dao have an admin? Im just making sure we are leaving a migration path here.

Reply:
Well, nm, this is the execute_admin msg path, so if we break exectuting admin messages for raw dao that doesnt have admin in the first place, that shouldnt matter.

Reply:
But actually looking down further, this could break nominating an admin: https://github.com/DA0-DA0/dao-contracts/pull/344/commits/01ca0d5cc7db96d6e1c95d0381d7180f819527c5#diff-cc5c48658f12e554fb961f3f264fb1efd28ebd41512da1f65fa27a0339d7df9aR201

Reply:
Can do something like this if RAW DAO wants to update to the latest contracts: https://github.com/DA0-DA0/dao-contracts/blob/zeke/cw20-stake-migration/contracts/cw20-stake/src/contract.rs#L451-L467